src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -1999,25 +1999,21 @@ internal override BaseClientSessionTransportManager CreateClientSessionTransport
         /// StartSSHProcess
         /// </summary>
         /// <returns></returns>
-        internal System.Diagnostics.Process StartSSHProcess(
+        internal int StartSSHProcess(
             out StreamWriter stdInWriterVar,
             out StreamReader stdOutReaderVar,
             out StreamReader stdErrReaderVar)
         {
             string filePath = string.Empty;
-#if !UNIX
             var context = Runspaces.LocalPipeline.GetExecutionContextFromTLS();
             if (context != null)
             {
-                var cmdInfo = context.CommandDiscovery.LookupCommandInfo("ssh.exe", CommandOrigin.Internal) as ApplicationInfo;
+                var cmdInfo = context.CommandDiscovery.LookupCommandInfo("ssh", CommandOrigin.Internal) as ApplicationInfo;
```

Comment:
I was hoping to simply the code a bit between platforms.  But I see your point and will fix.